cylc/uiserver/resolvers.py

```diff
@@ -321,6 +324,76 @@ async def play(cls, workflows, args, workflows_mgr, log):
         await workflows_mgr.scan()
         return response
 
+    @classmethod
+    async def enqueue(cls, stream, queue):
+        async for line in stream:
+            await queue.put(line.decode())
+
+    @classmethod
+    async def cat_log(cls, workflow: Tokens, log, info, task=None, file=None):
+        """Calls `cat log`.
+
+        Used for log subscriptions.
+        """
+        full_workflow = workflow
+        if file and task:
+            full_workflow = Tokens(f"{workflow.workflow_id}//{task}")
+        cmd = ['cylc', 'cat-log']
+        if file:
+            cmd += ['-f', file]
+        else:
+            cmd += ['-m', 't']
```

Comment:
I think we should always `tail` the logs?

```suggestion
        cmd += ['-m', 't']
```